File: ui/mirage/factories/feature.js
/**
 * Copyright IBM Corp. 2016, 2025
 * SPDX-License-Identifier: BUSL-1.1
 */

import { Factory } from 'miragejs';

export default Factory.extend({
  feature_flags() {
    return []; // VAULT_CLOUD_ADMIN_NAMESPACE
  },
});

File: sdk/helper/license/feature.go
// Copyright IBM Corp. 2016, 2025
// SPDX-License-Identifier: MPL-2.0

package license

// Features is a bitmask of feature flags
type Features uint64

const FeatureNone Features = 0

func (f Features) HasFeature(flag Features) bool {
	return false
}
